tests/ApiProblemTest.php

```diff
@@ -30,6 +30,12 @@ public function testConstructor()
         $this->assertEquals("URI", $problem->getType());
     }
 
+    public function testContentType()
+    {
+        $this->assertEquals('application/problem+json', ApiProblem::CONTENT_TYPE_JSON);
+        $this->assertEquals('application/problem+xml', ApiProblem::CONTENT_TYPE_XML);
+    }
```

Comment:
I'm not sure this test is needed...  It can't fail without a critical bug in the PHP engine. :smile:

Reply:
But it also doesn't hurt to be there and might act as a guard against a removal of these constants; but if you want I can remove it.

Reply:
Let's go ahead and remove it.  Thanks.